lib/scoped_search/query_builder.rb

```diff
@@ -402,7 +402,11 @@ def reflection_conditions(reflection)
 
       def to_ext_method_sql(key, operator, value, &block)
         raise ScopedSearch::QueryNotSupported, "'#{definition.klass}' doesn't respond to '#{ext_method}'" unless definition.klass.respond_to?(ext_method)
-        conditions = definition.klass.send(ext_method.to_sym,key, operator, value) rescue {}
+        begin
+          conditions = definition.klass.send(ext_method.to_sym, key, operator, value) 
+        rescue Error => e
```

Comment:
I didn't find this class definition, did you mean `StandardError` perhaps?

Reply:
Oops, you're right.